### PR TITLE
JohnTresadern-belfast

### PR DESCRIPTION
**Pull request by JohnTresadern**
This pull request adds a file about Belfast, Northern Ireland

### CODE REVIEW

File: JohnTresadern-belfast.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Belfast, Northern Ireland
+
+## Places to Eat
+- Mourne Seafood Bar for :fish:
+- The Morning Star for steak and :beers:
+- Macau for chinese food
+
+## Places to Visit
+- Crumlin Gaol
```
Comment (unixdaddy) [inline JohnTresadern-belfast.md:9]: to add a list use '-'

File: JohnTresadern-belfast.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Belfast, Northern Ireland
+
+## Places to Eat
+- Mourne Seafood Bar for :fish:
+- The Morning Star for steak and :beers:
+- Macau for chinese food
+
```
Comment (robertrankin) [inline JohnTresadern-belfast.md:7]: Fancy!

File: JohnTresadern-belfast.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Belfast, Northern Ireland
+
+## Places to Eat
+- Mourne Seafood Bar for :fish:
```
Comment (unixdaddy) [inline JohnTresadern-belfast.md:4]: I prefer steak  ;-)

File: JohnTresadern-belfast.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+- The Morning Star for steak and :beers:
+- Macau for chinese food
+
+## Places to Visit
```
Comment (unixdaddy) [inline JohnTresadern-belfast.md:8]: Add some indoor locations as as i heard it always rains in belfast

**Review by unixdaddy — CHANGES_REQUESTED** [review]
i have no idea but this looks cool to me

**Review by robertrankin — APPROVED** [review]

File: JohnTresadern-belfast.md (hunk 3 above)
Comment (JohnTresadern) [reply JohnTresadern-belfast.md:4]: they have swordfish steak

File: JohnTresadern-belfast.md (hunk 4 above)
Comment (JohnTresadern) [reply JohnTresadern-belfast.md:8]: No it only rains twice a year - Jan - June and then July to Dec

**Pull request merged**